front_end/html/view_pp.html

```diff
@@ -0,0 +1,27 @@
+{% if course_basics["exists"] %}
+    <h2>Pair Programming Assignments</h2>
```

Comment:
Please change the header to "Pair programming assignments" (only capitalize the first word)

Reply:
Just to maintain consistency with what we are doing on other pages.